uavcan/internet/2500.OutgoingUDPPacket.uavcan

```diff
@@ -0,0 +1,121 @@
+#
+# This message carries UDP packets from a node on the local bus to a remote host on
+# the Internet or a LAN.
+#
+# Any node can broadcast a message of this type.
+#
+# All nodes that are capable of communication with the Internet or a LAN should subscribe to messages
+# of this type and forward the payload to the indicated host via one or multiple UDP datagrams.
+# Such nodes will be referred to as modem nodes.
+#
+# It is expected that some systems will have more than one modem node available.
+# Each modem node is supposed to forward every message it sees, which will naturally create
+# some degree of modular redundancy and fault tolerance. The remote host should therefore be able to
+# properly handle possibly duplicating messages from different source addresses, in addition to
+# possible duplications introduced by the UDP protocol itself. There are at least two obvious
+# strategies that can be employed by the remote host:
+#
+#   - Accept only the first message, ignore duplicates. This approach requires that the UDP stream
+#     should contain metadata necessary for the remote host to determine the source and ordering of
+#     each received datum. This approach works best for periodic data, such as telemetry, where the
+#     sender does not expect any responses.
+#
+#   - Process all messages, including duplicates. This approach assumes that the remote host acts
+#     as a server, processing all received requests and providing responses to each. This arrangement
+#     implies that the client may receive duplicated responses. It is therefore the client's
+#     responsibility to resolve the possible ambiguity. An obvious solution is to accept the first
+#     arrived response and ignore the later ones.
+#
+# Applications are free to choose whatever redundancy management strategy works best for them.
+#
+# If the source node expects that the remote host will send some data back, it must explicitly notify
+# the modem nodes about this, so that they could prepare to perform reverse forwarding when the
+# expected data arrives from the remote host. The technique of reverse forwarding is known in
+# networking as IP Masquerading, or (in general) Network Address Translation (NAT). The notification
+# is performed by means of setting one of the corresponding flags defined below.
+#
+# In order to be able to match datagrams received from remote hosts and the local nodes they should
+# be forwarded to, modem nodes are required to keep certain metadata about outgoing datagrams. Such
+# metadata is stored in a data structure referred to as NAT table, where every entry would normally
+# contain at least the following fields:
+#   - The local UDP port number that was used to send the outgoing datagram from.
```

Comment:
Yes. This is a normal practice for UDP masquerading. Also, see this: https://en.wikipedia.org/wiki/UDP_hole_punching